### ISSUE TITLE
Story Maps: "Add new chapter" button fails accessibility guidelines

### ISSUE THREAD

**Issue by DerekCaelin** [techmatters/terraso-web-client#2806]
Not actually a bug (code working as designed) but an issue. 

<img width="360" height="156" alt="Image" src="https://github.com/user-attachments/assets/cd172845-6c80-4921-883a-313911b58fd0" />

Need:
- [ ] Design alternative solution
- [ ] Implement design
